examples/flava/coco_zero_shot.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import argparse
+
+import torch
+from data.transforms import (
+    default_image_pretraining_transforms,
+    default_text_transform,
+    default_torchvision_transforms,
+)
+from torch import nn
+from torch.utils.data import DataLoader
+from torchmultimodal.models.flava import flava_model_for_pretraining
+from torchvision.datasets import CocoCaptions
+
+
+def compute_recall(similarity_scores: torch.Tensor, k: int = 5):
+    dataset_size = similarity_scores.size(0)
+    targets = torch.arange(dataset_size)
+    _, topk_idx = torch.topk(similarity_scores, k, dim=1)
+
+    recall = 0
+    for index in range(dataset_size):
+        recall += index in topk_idx[index]
+
+    recall = recall / dataset_size
+    return recall
+
+
+def transform(image, target):
+    _, image_transform = default_image_pretraining_transforms()
+    _, image_transform = default_torchvision_transforms()
```

Comment:
You probably want to use `default_image_pretraining_transforms` only?

Reply:
yeah thats leftover